### ISSUE TITLE
Is the project abandoned?

### ISSUE THREAD

**Issue by elchininet** [joonhocho/tscpaths#33]
Hi @joonhocho, cc: @jonkwheeler, @adam187,

I’m seeing that there are issues opened since 2018 and pull requests from 2019. Any chance that you have decided to no longer maintain this project anymore?

In that case, it would be good to state it in the README, I use this package in all my libraries and I would like to address some of the issues that it has. I would be also open to being a maintainer of the project or taking it over if you are no longer interested.

Regards

**Comment by jonkwheeler** [comment]
@elchininet yes the project was abandoned. I forked it and made my own. I use this for all my libraries, and I haven't abandoned it. https://github.com/jonkwheeler/tsconfig-replace-paths

**Comment by elchininet** [comment]
> @elchininet yes the project was abandoned. I forked it and made my own. I use this for all my libraries, and I haven't abandoned it. https://github.com/jonkwheeler/tsconfig-replace-paths

Perfect, I’ll migrate my projects to `tsconfig-replace-paths` then.

Thanks

**Comment by elchininet** [comment]
@jonkwheeler,

I’ve migrated my libraries to `tsconfig-replace-paths`. No problems so far.

Thanks.

**Comment by BlackGlory** [comment]
Try this [LeDDGroup/typescript-transform-paths](https://github.com/LeDDGroup/typescript-transform-paths)

**Comment by Xunnamius** [comment]
Perhaps unfortunately, [LeDDGroup/typescript-transform-paths](https://github.com/LeDDGroup/typescript-transform-paths) requires [monkey-patching the TypeScript compiler](https://github.com/LeDDGroup/typescript-transform-paths#3-usage), though the feature set is neat.

[`tsconfig-replace-paths`](https://github.com/jonkwheeler/tsconfig-replace-paths) works great, though `extends` support is broken (as of October 20, 2021) until the PR fixing it get merged in.

EDIT: These days, I use [babel-plugin-transform-rewrite-imports](https://www.npmjs.com/package/babel-plugin-transform-rewrite-imports), which provides a superset of tscpaths/tsconfig-replace-paths functionality and generates more consistent and accurate output (especially for type-only dynamic imports in `.d.ts` files).

**Comment by bjiusc** [comment]
I've been using https://www.npmjs.com/package/tsc-alias which has more adoption and is actively maintained
